### PR TITLE
fix(repo): make github token input optional

### PR DESCRIPTION
**Pull request by danadajian**
### :pencil: Description
- Never realized we could make the `github_token` optional and default to `${{ github.token }}`, but we can!

### CODE REVIEW

**Review by aschwenn — APPROVED** [review]
awesome!

File: .github/workflows/manage-merge-queue.yml [hunk 1]
```diff
@@ -15,12 +15,7 @@ jobs:
...
         uses: actions/checkout@v3
-        with:
-          ref: ${{ github.event.pull_request.head.ref }}
-          repository: ${{ github.event.pull_request.head.repo.full_name }}
-          token: ${{ secrets.GH_PERSONAL_ACCESS_TOKEN }}
 
       - uses: ./
         with:
           helper: manage-merge-queue
-          github_token: ${{ secrets.GH_PERSONAL_ACCESS_TOKEN }} # must have repo:write permission
```
[the remark this answers is not in the document]
Comment (danadajian) [reply .github/workflows/manage-merge-queue.yml:26]: The default token has repo:write permission by default. Do you still think this info is valuable? I'm good either way

[the remark this answers is not in the document]
Comment (danadajian) [reply .github/workflows/manage-merge-queue.yml:26]: I started noting these to explicitly call out when the default `GITHUB_TOKEN` is not enough to use the helper

[the remark this answers is not in the document]
Comment (aschwenn) [reply .github/workflows/manage-merge-queue.yml:26]: i guess i was thinking within a forking context, where it's read-only. not a big deal either way

[the remark this answers is not in the document]
Comment (danadajian) [reply .github/workflows/manage-merge-queue.yml:26]: Even forks don't need to worry passing a special token for repo:write access

**Pull request merged**